### ISSUE TITLE
Can the depth data file use ETOPO2022's geotiff format?

### ISSUE THREAD

**Issue by DickWhite** [HandsomeAndy/GPU-comcot#2]

**Comment by AndybnACT** [comment]
Yes, I think supporting any non text-based (ascii) format is essential for any simulation software. However, I think I do not have time to do it for now. Please consider transforming the file to the text-based .xyz file with whatever tool available and run the comcot. Or create a PR to make this nice feature works ;)
